src/frontend/komponenter/Fagsak/Vilkår/BehandleVilkårSkjema.tsx

```diff
@@ -82,6 +82,21 @@ const BehandlingVilkårSkjema: React.FunctionComponent<IBehandlingVilkårSkjema>
                     });
                 }}
             />
+
+            <br/>
+            
+            <TextareaControlled 
+                label={'Begrunnelse'}
+                maxLength={0}
```

Comment:
Why 0?

Reply:
0 means no limitation to the length. Should we enforce a maximum length of the content?

Reply:
Ok, not very intuitive, but okey! 👍 No limitations in the first version at least :)